@xen-orchestra/lite/src/App/Pools/PoolNetwork.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import { FormattedMessage } from 'react-intl'
+import { Map } from 'immutable'
+import { withState } from 'reaclette'
+
+import { Network, ObjectsByType, Pif } from '../../libs/xapi'
+
+interface ParentState {
+  objectsByType: ObjectsByType
+}
+
+interface State {}
+
+interface Props {
+  poolId: string
+}
+
+interface ParentEffects {}
+
+interface Effects {}
+
+interface Computed {
+  managementPIFs?: Map<string, Pif>
+  networks?: Map<string, Network>
+  objectsFetched: boolean
+  PIFs?: Map<string, Pif>
+}
+
+const Table = styled.table`
+  border: 1px solid #333;
+  td  {
+    border: 1px solid #333;
+  }
+  thead {
+    background-color: #333;
+    color: #fff;
+  }
+`
+
+const PoolNetwork = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
+  {
+    computed: {
+      managementPIFs: state => state.PIFs?.filter(pif => pif.management),
+      networks: (state, props) =>
+        state.objectsFetched
+          ? state.objectsByType.get('network')?.filter(network => network.$pool.$id === props.poolId)
+          : undefined,
+      objectsFetched: state => state.objectsByType !== undefined,
```

Comment:
There should already be an `objectsFetched` in the state: https://github.com/vatesfr/xen-orchestra/blob/3bdbdf2d1fd573fb0527fce066be79ab348ef3d5/%40xen-orchestra/lite/src/App/index.tsx#L66.